wandb/sdk/artifacts/artifact.py

```diff
@@ -531,6 +531,27 @@ def source_collection(self) -> ArtifactCollection:
     def type(self) -> str:
         """The artifact's type. Common types include `dataset` or `model`."""
         return self._type
+    
+    @property
+    def url(self):
+        """
+        Constructs the URL of the artifact.
+
+        Returns:
+            str: The URL of the artifact.
+        """
+        if self.collection.is_sequence():
+            return f"{self._client.app_url}{self._entity}/{self._project}/artifacts/{self._type}/{self._name}"
+        else:
+            if self._project.startswith("wandb-registry-"):
+                org = InternalApi()._fetch_orgs_and_org_entities_from_entity(
+                    self._entity
+                )[0]
+                return f"{self._client.app_url}orgs/{org.display_name}/registry/{self._type}?selectionPath={self._entity}%2F{self._project}%2F{self.collection.name}&view=membership&version={self._version}"
```

Comment:
selectionPath={self._entity} can also be selectionPath={org.entity_name}, not sure if there's any preference?

Reply:
`self._entity` should be fine